src/shared/rpc/methods/handle-bitcoin-contract-method.ts

```diff
@@ -0,0 +1,27 @@
+import { DefineRpcMethod, RpcRequest, RpcResponse } from '@btckit/types';
+import { AllowAdditionaProperties } from '@btckit/types/dist/types/utils';
+
+export interface HandleBitcoinContractResponseParams extends AllowAdditionaProperties {
+  bitcoinContractOffer: string;
+  counterpartyWalletURL: string;
+  counterpartyWalletName: string;
+  counterpartyWalletIcon: string;
+}
+
+export interface HandleBitcoinContractResponseBody extends AllowAdditionaProperties {
+  id: string;
+  contractID: string;
```

Comment:
Can this be `contractId` and `txId` on L15?

Reply:
Sure!